tools/install.py

```diff
@@ -133,6 +133,8 @@ def files(action):
   action(['src/node.stp'], 'share/systemtap/tapset/')
 
   action(['deps/v8/tools/gdbinit'], 'share/doc/node/')
+  action(['deps/v8/tools/lldbinit'], 'share/doc/node/')
+  action(['deps/v8/tools/lldb_commands.py'], 'share/doc/node/')
```

Comment:
Why do we install programs in `shared/doc`? https://github.com/nodejs/node/pull/2123 did the same for `gdbinit`, and it seems it made sense for @ofrobots and @bnoordhuis, so I'm probably just missing something.